pilot/pkg/model/validation.go

```diff
@@ -28,7 +28,6 @@ import (
 	"github.com/gogo/protobuf/proto"
 	"github.com/gogo/protobuf/types"
 	"github.com/hashicorp/go-multierror"
-
```

Comment:
We usually separate the 3rd party from project's import with an empty line

Reply:
that was one of my formatting plugins.. fixed